src/object_store/src/object/opendal_engine/azblob.rs

```diff
@@ -38,6 +38,7 @@ impl OpendalObjectStore {
         builder.account_key(&account_key);
         let op: Operator = Operator::new(builder)?
             .layer(RetryLayer::default())
+            .layer(LoggingLayer::default())
```

Comment:
It's better to implement something like https://opendal.apache.org/docs/rust/opendal/struct.OperatorBuilder.html#examples

But this PR is Ok to go, we can refactor this part later.